src/main/java/net/snowflake/ingest/streaming/internal/DropChannelRequestInternal.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2024 Snowflake Computing Inc. All rights reserved.
+ */
+
+package net.snowflake.ingest.streaming.internal;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import net.snowflake.ingest.streaming.DropChannelRequest;
+import net.snowflake.ingest.utils.Utils;
+
+/** Class used to serialize the {@link DropChannelRequest} */
+class DropChannelRequestInternal implements StreamingIngestRequest {
+  @JsonProperty("request_id")
+  private String requestId;
+
+  @JsonProperty("role")
+  private String role;
+
+  @JsonProperty("channel")
+  private String channel;
+
+  @JsonProperty("table")
+  private String table;
+
+  @JsonProperty("database")
+  private String database;
+
+  @JsonProperty("schema")
+  private String schema;
+
+  @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
same as OpenChannelRequestInternal, should be ok to remove this NON_NULL and do the null check at the callsite instead?

Reply:
Discussed offline.